packages/gamut-labs/src/landingPage/PageHeroMedia.tsx

```diff
@@ -0,0 +1,44 @@
+import { Column, Video } from '@codecademy/gamut';
+import { breakpoints } from '@codecademy/gamut-styles';
+import styled from '@emotion/styled';
+import { omit } from 'lodash';
+import React from 'react';
+
+import { MediaProps } from './PageHero';
+
+const RightColumn = styled(Column)`
+  margin-left: 1rem;
+
+  @media only screen and (max-width: ${breakpoints.sm}) {
+    display: none;
+  }
+`;
+
+const VideoColumn = styled(Column)`
+  margin-left: 1rem;
+`;
+
+const Image = styled.img`
+  width: 100%;
+`;
+
+const StyledVideo = styled(Video)`
+  align-self: center;
+`;
+
+export const PageHeroMedia: React.FC<MediaProps> = (media) => {
+  if (media.type === 'image') {
+    return (
+      <RightColumn size={3}>
+        <Image src={media.src} alt={media.alt} />
+      </RightColumn>
+    );
+  } else if (media && media.type === 'video') {
```

Comment:
This check is unnecessary because the prop declares it must exist, right?

```suggestion
  } else if (media.type === 'video') {
```

Reply:
true I missed that when cleaning up